http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20240117_create_app_chart_version_table.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="ladieu" id="create_app_chart_version_table">
+        <createTable tableName="APP_CHART_VERSION">
+            <column name="chartName" type="VARCHAR(254)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="chartVersion" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="appTargetVersion" type="VARCHAR(254)">
```

Comment:
nit, `appVersion` instead of `appTargetVersion`?

Reply:
I had doubts about `appTargetVersion`...thanks for the nudge.  Fixed in d31af9527337378aeecdcb19bc44a3685143de3f